### PR TITLE
fix: Fetch all exhibitors after rendering exhibition view page.

### PR DESCRIPTION
**Pull request by sachinchauhan2889**
In continue of #6891

### CODE REVIEW

**Comment by iamareebjamal** [comment]
What's remaining?

File: app/routes/public/exhibition/view.js [hunk 1]
```diff
@@ -17,14 +17,16 @@ export default class ViewRoute extends Route {
     });
   }
 
-  async afterModel(model) {
+  afterModel(model) {
```
[the remark this answers is not in the document]
Comment (sachinchauhan2889) [reply app/routes/public/exhibition/view.js:20]: Sir, twitter widget error. Please visit deployment after my first commit and check terminal.

[the remark this answers is not in the document]
Comment (iamareebjamal) [reply app/routes/public/exhibition/view.js:20]: But why fetching exhibitors in it?

[the remark this answers is not in the document]
Comment (sachinchauhan2889) [reply app/routes/public/exhibition/view.js:19]: resolved on call

**Pull request merged**